osu.Game.Rulesets.Taiko/Mods/TaikoModEasy.cs

```diff
@@ -1,12 +1,40 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using System;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Beatmaps;
+using osu.Game.Graphics;
 using osu.Game.Rulesets.Mods;
 
 namespace osu.Game.Rulesets.Taiko.Mods
 {
-    public class TaikoModEasy : ModEasy
+    public class TaikoModEasy : Mod, IApplicableToDifficulty, IApplicableFailOverride
     {
-        public override string Description => @"Beats move slower, less accuracy required, and three lives!";
+        public override string Name => "Easy";
+        public override string Acronym => "EZ";
+        public override string Description => @"Beats move slower, less accuracy required";
```

Comment:
for consistency with the old one

```suggestion
        public override string Description => @"Beats move slower, and less accuracy required!";
```